app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
   before_action :require_signin_permission!
 
-  protect_from_forgery with: :exception
+  rescue_from ActionController::ParameterMissing do |e|
+    render json: { error: { message: e.message } }, status: 400
+  end
+
+  rescue_from ActiveRecord::RecordNotFound do |e|
+    render json: { error: { message: e.message } }, status: 404
+  end
```

Comment:
Yeah, I don't think it wraps it in test - I recall writing tests expecting that exception. Though try it on rails in dev and prod envs to see. 

There's a whole bunch of things wrapped around: https://github.com/rails/rails/blob/fe1f4b2ad56f010a4e9b93d547d63a15953d9dc2/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb